src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerdeletionservice/repository/MovementsRepositoryTest.kt

```diff
@@ -40,4 +40,17 @@ class MovementsRepositoryTest {
     val movements: List<Movement> = repository.getMovementsByOffenders(listOf("A6676RS"), listOf(), false, false)
     assertThat(movements).extracting<String>(Movement::fromAgency).containsOnly("BMI", "LEI")
   }
+
+  @Test
+  fun `can retrieve deceased movement by offenders`() {
```

Comment:
You could make this a `@ParameterizedTest` passing in the prison number and expected results which would remove the duplicated assertions.